packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -159,7 +160,27 @@ export class SummarizerNode implements IRootSummarizerNode {
 			}
 		}
 
-		const result = await this.summarizeInternalFn(fullTree, true, telemetryContext);
+		// This assert is the same the other 0a1x1 assert `isSummaryInProgress`, the only difference is that typescript
+		// complains if this assert isn't done this way
+		assert(
+			this.wipReferenceSequenceNumber !== undefined,
+			"Same as assert 0x1a1 - Summarize should not be called when tracking the summary",
+		);
+		const incrementalSummaryContext: IIncrementalSummaryContext | undefined =
+			this._latestSummary !== undefined
+				? {
+						wipSummarySequenceNumber: this.wipReferenceSequenceNumber,
+						lastAckedSummarySequenceNumber: this._latestSummary.referenceSequenceNumber,
+						summaryPath: this._latestSummary.fullPath.path,
```

Comment:
This path contains the id of the data store and the DDS which should not be leaked to layers below them. Ideally, a layer should not know its own id.
This is important for channel unification work and there has been a lot of work to remove these kind of leakages. Some still exist which have to be fixed but we should not be adding more dependencies.

@vladsud as FYI

Reply:
Copying this comment into the summarizerNode itself.

Reply:
Can you create an issue to follow up on this.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/3969